io.gdevelop.ide.yml

```diff
@@ -0,0 +1,50 @@
+# This is the Flatpak manifest for GDevelop.
+app-id: io.gdevelop.ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  # - --socket=wayland # Activate this when GDevelop uses Electron 12+
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.Notifications
+modules:
+  - name: gdevelop-ide
+    buildsystem: simple
+
+    sources:
+      - type: file
+        url: https://gdevelop-releases.s3.amazonaws.com/master/commit/50326e4d2147e0b7efca02ef062dba97831cd3d7/gdevelop_5.0.130_amd64.deb
```

Comment:
Is there some sort of universal "give me the latest deb url"? if yes, https://github.com/flathub/flatpak-external-data-checker/ could be used to automate pull requests for new releases.